mava/wrappers/jumanji.py

```diff
@@ -112,8 +112,9 @@ def _add_agent_ids(
         # just need to check that this doesn't affect timing
```

Comment:
Can you remove these comments above here. I had another look and it's not possible to do this unfortunately